Tests/MessagingPushAPN/APITest.swift

```diff
@@ -18,6 +18,12 @@ class MessagingPushAPNAPITest: UnitTest {
     func test_allPublicFunctions() throws {
         try skipRunningTest()
 
+        // This is the `initialize()` function that's available to Notification Service Extension and not available
+        // to other targets (such as iOS).
+        // You should be able to uncomment the initialize() function below and should get compile errors saying that the
+        // function is not available to iOS.
+        // MessagingPush.initialize(writeKey: "") { (config: inout MessagingPushConfigOptions) in }
```

Comment:
can we have `available` check on it instead of commenting it out?

Reply:
I don't think XCode will allow calling because it is only available for `iOSApplicationExtension`. And that's probably the reason why we have it the [same way in `main` today](https://github.com/customerio/customerio-ios/blob/216341c932d29caf334da53df3b4ab37b97bc94a/Tests/Tracking/APITest.swift#L43). Do you have any other ideas that you would like to share?